doc/source/config.rst

```diff
@@ -37,6 +37,46 @@ several config files and pick from them using `-c`. If you only wish to control
 the specific tests that are to be run (and not their parameters) then using
 `-s` or `-t` on the command line may be more appropriate.
 
+Also you can configure bandit via
+`pyproject.toml <https://www.python.org/dev/peps/pep-0518/>`_ file. In this
+case you would explicitly specify the path to configuration via `-c` too.
+For example:
+
+.. code-block:: TOML
+
+    [tool.bandit.profiles.test_1]
```

Comment:
These profile examples are a deprecated use of configuration.  We should give non-legacy examples of ways to configure here.

`
[config]	WARNING	Config file 'pyproject.toml' contains deprecated legacy config data. Please consider upgrading to the new config format. The tool 'bandit-config-generator' can help you with this. Support for legacy configs will be removed in a future bandit version.`